docs/modules/data-structures/pages/map-config.adoc

```diff
@@ -33,9 +34,22 @@ For details on map backups, refer to xref:backing-up-maps.adoc[].
 
 For details on in-memory format, refer to xref:setting-data-format.adoc[].
 
-== Modifying the Default Configuration
+== The Default (Fallback) Map Configuration
+When a map is created, if the map name matches an entry in hazelcast.xml/yaml file, the values in the matching entry are used to overwrite initial values
+specified in <<map-configuration-defaults,Map Configuration Defaults>> section.
 
-You can create a default configuration for all maps for your environment by modifying the map configuration block named "default" in your `hazelcast.xml`/`hazelcast.yaml` file. In the following example, we set expiration timers for map entries. Map entries that are idle for an hour will be marked as eligible for removal if the cluster begins to run out of memory. Any map entry older than six hours will be marked as eligible for removal. 
+This path also includes wildcard matches, meaning entries with “*” character
```

Comment:
```suggestion
Maps that do not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all maps, you can name your configuration as `default`. A user-defined default configuration applies to every map that does not have a specific custom map configuration defined with the map’s name. You can also use wildcards to associate your configuration with multiple map names. See the [configuration documentation](https://docs.hazelcast.com/hazelcast/5.5/configuration/using-wildcards) for more information about wildcards.
```

This is actually a snippet taken from Cloud's docs, and I think it's pretty well worded: https://docs.hazelcast.com/cloud/map-configurations